zip-0316.rst

```diff
@@ -38,8 +38,8 @@ Consumer
   that it is given.
 Sender
   A wallet or other software that can send transfers of assets, or other
-  consensus state side-effects defined in future. Senders are a subset of
-  Consumers.
+  side effects to consensus state that may be defined in future. Senders are a
+  subset of Consumers.
```

Comment:
```suggestion
  side effects to consensus state that may be defined in the future. Senders
  are a subset of Consumers.
```

Reply:
This means the same thing to me. Maybe it's a British/US English difference?